test/test_matmul_cuda.py

```diff
@@ -664,6 +665,7 @@ def test_float8_error_messages(self, device) -> None:
             )
 
     @unittest.skipIf(not PLATFORM_SUPPORTS_FP8 or IS_WINDOWS, f8_msg)
```

Comment:
Isn't this a redundant as `PLATFORM_SUPPORTS_FP8` should imply `SM90OrLater` at the moment?

Reply:
Oh interesting, `PLATFORM_SUPPORTS_FP8` also support (89) or later. Should ROWWISE MatMul's be redone to support 89 as well?

Reply:
So 4090 users are left out high and dry at the moment...